docs/user/migrating-to-v1.0.0.md

```diff
@@ -1,3 +1,32 @@
 # Migration to v1.0.0
 
-This release includes a breaking change **if you are upgrading from a version prior to `v1.0.0-beta.2`**. In the version `v1.0.0-beta.2` we introduced a change in MongoDB credentials generation that requires a migration hook that is only present from the version `v1.0.0-beta.2` to `v1.0.0-beta.4`. If you are upgrading Kubeapps from an older version please upgrade first to `v1.0.0-beta.4` to run the necessary migration job.
+This release includes several breaking changes that can make upgrading fairly
+difficult. Helm 2.10 and above is also now required to install Kubeapps.
+
+If you have difficulty upgrading to the v1.0.0 release, we recommend backing up
+any AppRepository objects (custom repositories) you may have added and perform a
```

Comment:
> any AppRepository objects (custom repositories)

`custom repositories` do not fully sound to me like the ones that the user created.  We might want to be more explicit by saying, `backup any AppRepository that you have created`

Reply:
it does say "you may have added", does that not cover it? The reason to use "custom repositories" in brackets is because users might not actually know what an AppRepository is if they've only added repos in the UI.

Reply:
Sounds good! :+1: